lib/api/href.rb

```diff
@@ -0,0 +1,107 @@
+module Api
+  class Href
+    # @param href [String] the href string
+    #
+    #   Supports the following forms:
+    #   - http://www.example.com/api
```

Comment:
I can't find any RDoc documentation on how to prevent this from being rendered as a link 😕

Reply:
Any ideas @durandom , @chrisarcand ?